spec/mutations/availability/create_availability_spec.rb

```diff
@@ -44,34 +48,74 @@ def query(status:)
-        expect(json['data']['createAvailability']['id']).to_not be(nil)
-        expect(json['data']['createAvailability']['userId']).to eq(@user.id.to_s)
-        expect(json['data']['createAvailability']['startDateTime']).to eq('1609489800')
-        expect(json['data']['createAvailability']['endDateTime']).to eq('1609504200')
-        expect(json['data']['createAvailability']['status']).to eq('open')
+        json = JSON.parse(response.body, symbolize_names: true)
+        expect(json[:data][:createAvailability][:id]).to_not be(nil)
+        expect(json[:data][:createAvailability][:userId]).to eq(@user.id.to_s)
+        expect(json[:data][:createAvailability][:startDateTime]).to eq('1609489800')
+        expect(json[:data][:createAvailability][:endDateTime]).to eq('1609504200')
+        expect(json[:data][:createAvailability][:status]).to eq('open')
       end
 
       it 'An availability can be created as fulfilled' do
         post '/graphql', params: { query: query(status: 1) }
-        json = JSON.parse(response.body)
+        json = JSON.parse(response.body, symbolize_names: true)
 
-        expect(json['data']['createAvailability']['id']).to_not be(nil)
-        expect(json['data']['createAvailability']['userId']).to eq(@user.id.to_s)
-        expect(json['data']['createAvailability']['startDateTime']).to eq(@start_dt.to_s)
-        expect(json['data']['createAvailability']['endDateTime']).to eq(@end_dt.to_s)
-        expect(json['data']['createAvailability']['status']).to eq('fulfilled')
+        expect(json[:data][:createAvailability][:id]).to_not be(nil)
+        expect(json[:data][:createAvailability][:userId]).to eq(@user.id.to_s)
+        expect(json[:data][:createAvailability][:startDateTime]).to eq(@start_dt.to_s)
+        expect(json[:data][:createAvailability][:endDateTime]).to eq(@end_dt.to_s)
+        expect(json[:data][:createAvailability][:status]).to eq('fulfilled')
       end
 
       it 'An availability can be created as cancelled' do
         post '/graphql', params: { query: query(status: 2) }
-        json = JSON.parse(response.body)
+        json = JSON.parse(response.body, symbolize_names: true)
 
-        expect(json['data']['createAvailability']['id']).to_not be(nil)
-        expect(json['data']['createAvailability']['userId']).to eq(@user.id.to_s)
-        expect(json['data']['createAvailability']['startDateTime']).to eq(@start_dt.to_s)
-        expect(json['data']['createAvailability']['endDateTime']).to eq(@end_dt.to_s)
-        expect(json['data']['createAvailability']['status']).to eq('cancelled')
+        expect(json[:data][:createAvailability][:id]).to_not be(nil)
+        expect(json[:data][:createAvailability][:userId]).to eq(@user.id.to_s)
+        expect(json[:data][:createAvailability][:startDateTime]).to eq(@start_dt.to_s)
+        expect(json[:data][:createAvailability][:endDateTime]).to eq(@end_dt.to_s)
+        expect(json[:data][:createAvailability][:status]).to eq('cancelled')
+      end
```

Comment:
Not likely that we'll ever create an availability w/ status of fulfilled or cancelled but doesn't hurt to have more thorough testing! @lkriffell